yarn.lock

```diff
@@ -2,71 +2,55 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0":
-  version "7.0.0"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity "sha1-BuKrGb21NThVWaq7W6WXKUgoAPg= sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA=="
-  dependencies:
-    "@babel/highlight" "^7.0.0"
-
-"@babel/code-frame@^7.8.3":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.8.3":
   version "7.8.3"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.8.3.tgz#33e25903d7481181534e12ec0a25f16b6fcf419e"
-  integrity "sha1-M+JZA9dIEYFTThLsCiXxa2/PQZ4= sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g=="
+  resolved "https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/@babel/code-frame/-/code-frame-7.8.3.tgz"
+  integrity "sha1-M+JZA9dIEYFTThLsCiXxa2/PQZ4=sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g== sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g=="
```

Comment:
What is the process for installing new packages now, or bumping versions? Does `yarn install x@1` still work?

Reply:
no.

Reply:
Ok, what is the process?

Reply:
ideally.

bazel run @nodejs//:yarn config set registry https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/
bazel run @nodejs//:yarn add x@1.

Reply:
It would be good to document this, right? https://github.com/dataform-co/dataform/blob/main/contributing.md#add-new-npm-dependencies